lua/nvim-web-devicons/icons-light.lua

```diff
@@ -1226,6 +1248,13 @@ local icons_by_file_extension = {
     cterm_color = "236",
     name = "OpenTypeFont",
   },
+  ["out"] = {
+    icon = "",
+    color = "#9F0500",
+    cterm_color = "196",
+    name = "Out",
+  },
+
```

Comment:
```suggestion
```

Reply:
Working on it.

Reply:
There. Every space should've been removed.